packages/language-service/ivy/template_target.ts

```diff
@@ -286,7 +286,7 @@ class TemplateTargetVisitor implements t.Visitor {
 
   visit(node: t.Node) {
     const {start, end} = getSpanIncludingEndTag(node);
-    if (!isWithin(this.position, {start, end})) {
+    if (end !== null && !isWithin(this.position, {start, end: end!})) {
```

Comment:
```suggestion
    if (end !== null && !isWithin(this.position, {start, end})) {
```

The non-null assertion _shouldn't_ be necessary here - unless TypeScript is doing something funny?

Reply:
Oh, this was from an earlier revision when I had the `end` as `number|null` and was just playing with things. Reverted.x`